entries/ad-layers-dfp/index.js

```diff
@@ -143,7 +145,8 @@ import './style.scss';
         }
       }
     }
-    return this.buildAd(args.slotName, args.path, args.sizes, args.targeting, args.sizeMapping, args.companion || false);
+
+    return this.buildAd(args.slotName, args.path, args.sizes, args.targeting, args.sizeMapping, args.companion || false, args.noDisplay || false);
```

Comment:
would `??` make more sense here than `||`? We're trying to define a default value, right?

Reply:
You're right - thanks!